replications/internal/queue_management.go

```diff
@@ -143,3 +148,91 @@ func (qm *durableQueueManager) CurrentQueueSizes(ids []platform.ID) (map[platfor
 
 	return sizes, nil
 }
+
+// StartReplicationQueues updates the durableQueueManager.replicationQueues map, fully removing any partially deleted
+// queues (present on disk, but not tracked in sqlite), opening all current queues, and logging info for each.
+func (qm *durableQueueManager) StartReplicationQueues(trackedReplications map[platform.ID]int64) error {
+	errOccurred := false
+
+	for id, size := range trackedReplications {
+		// Re-initialize a queue struct for each replication stream from sqlite
+		queue, err := durablequeue.NewQueue(
+			filepath.Join(qm.queuePath, id.String()),
+			size,
+			durablequeue.DefaultSegmentSize,
+			&durablequeue.SharedCount{},
+			durablequeue.MaxWritesPending,
+			func(bytes []byte) error {
+				return nil
+			},
+		)
+
+		if err != nil {
+			qm.logger.Error("failed to initialize replication stream durable queue", zap.Error(err))
+			errOccurred = true
```

Comment:
I think you need a `continue` after this to skip the rest of the logic in the loop for the failed queue